tests/plugins/test_settings.py

```diff
@@ -138,3 +176,277 @@ def test_load_plugin_config_with_env_var(
         getattr(context.plugins, STRING_PARAMETER_NAME)
         == STRING_PARAMETER_ENV_VAR_VALUE
     )
+
+
+def test_conda_config_with_string_settings(condarc_plugin_manager, tmp_path, conda_cli):
+    """
+    Ensure that string parameter types work correctly as a plugin setting
+    """
+    condarc = tmp_path / "condarc"
+    out, err, _ = conda_cli(
+        "config",
+        "--file",
+        condarc,
+        "--set",
+        f"plugins.{STRING_PARAMETER_NAME}",
+        "env_var_value",
```

Comment:
Personally I like to do something like this to help keep the arguments tightly associated:

```suggestion
        f"--file={condarc}",
        *("--set", f"plugins.{STRING_PARAMETER_NAME}", "env_var_value"),
```

Reply:
I see where you're coming from, but I would rather not do this.